src/features/game/events/landExpansion/ironMine.ts

```diff
@@ -0,0 +1,101 @@
+import Decimal from "decimal.js-light";
+import cloneDeep from "lodash.clonedeep";
+import { IRON_MINE_STAMINA_COST } from "../../lib/constants";
+import { trackActivity } from "../../types/bumpkinActivity";
+import { GameState, LandExpansionRock } from "../../types/game";
+import { replenishStamina } from "./replenishStamina";
+
+export type LandExpansionIronMineAction = {
+  type: "ironRock.mined";
```

Comment:
Should this just be `iron.mined`?

Reply:
Not really, we're using iron.mined for the current flow.. 
Land Expansion will be `ironRock.mined`, `stoneRock.mined`, `goldRock.mined` at least until we get rid of the other events.